app/Http/Controllers/ContactController.php

```diff
@@ -2,42 +2,30 @@
 
 namespace App\Http\Controllers;
 
-use App\Http\Controllers\Controller;
-use App\Models\Contact;
-use Illuminate\Http\Request;
+use App\Contact;
+use App\Http\Requests\ContactRequest;
+use App\Services\ContactService;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Response;
 
 class ContactController extends Controller
 {
-    
+
     public function index()
     {
         //
     }
-    
-    public function store(ContactRequest $request)
+
+    public function store(ContactRequest $request): JsonResponse
     {
         $parameters = $request->validated();
 
-        $contacts = [];
```

Comment:
This Store method was trying to store records **only** in the `contacts` a no in the `custom attribute table`. Since this required a little more login, A new `ContactService` class was added.